logfire/_internal/config.py

```diff
@@ -1304,11 +1334,37 @@ def from_token(cls, token: str, session: requests.Session, base_url: str) -> Sel
         )
 
     @classmethod
-    def _get_user_token(cls, logfire_api_url: str) -> str:
-        if DEFAULT_FILE.is_file():  # pragma: no branch
+    def _get_token_data(cls, logfire_api_url: str | None = None) -> tuple[str, str]:
```

Comment:
```suggestion
    def _get_user_token_data(cls, logfire_api_url: str | None = None) -> tuple[str, str]:
```

Reply:
this class is mostly about write tokens, so i'm not even sure this method should be here, but the name and the docs really need to emphasise the difference